css/app.css

```diff
@@ -113,10 +117,10 @@ img {
 }
 
 #tabs-list li {
+  align-items: center;
   display: flex;
   flex-direction: row;
-  align-items: center;
-  height: 42px;
+  height: var(--tab-height, 42px);
```

Comment:
For my info: why is `var(--tab-height);` insufficient? Why do you need the backup value here?

Reply:
It still works withoug it, it just seemed more robust to me in case it gets deleted later or something. Happy to remove if you like.